README.md

```diff
@@ -44,21 +44,21 @@
 
 To view a live example, **[click here](https://ashutosh1919.github.io/)**
 
-# Clone And Use 📋
+# Clone and use 📋
```

Comment:
```suggestion
# Clone and Use 📋
```

Reply:
Its generally a better practice to use sentence case rather than Title Case for readability reasons